client/src/partials/cash/modals/transfer.modal.html

```diff
@@ -0,0 +1,88 @@
+<form name="TransferForm" ng-submit="CashTransferModalCtrl.submit(TransferForm.$invalid)" novalidate>
+
+  <div class="modal-header">
+    <h4>{{ "CASH.VOUCHER.CASHBOXES.TRANSFER" | translate }}</h4>
+  </div>
+
+  <div class="modal-body">
+
+    <p class="text-danger text-center" ng-if="CashTransferModalCtrl.cashBoxIdMissed">
```

Comment:
Why not just use `CashTransferModalCtrl.cashBox`?  For example:
```html
<p class="text-danger text-center" ng-hide="CashTransferModalCtrl.cashBox">
  <span class="glyphicon glyphicon-alert"></span>
 {{ "CASH.MODALS.MISSING_CASH_BOX_ID" | translate }}
</p>
```
You can do the same thing for the `uib-alert`.
```html
<uib-alert type="info" ng-show="CashTransferModalCtrl.cashBox">
  <!-- ... code ... -->
</uib-alert>
```